posthog/models/entity.py

```diff
@@ -69,6 +69,19 @@ def equals(self, other) -> bool:
 
         return True
 
+    """ Checks if this entity is a generalized version of other"""
```

Comment:
Docstrings are added mostly _in_ the function, e.g. how it is above - should they be here?

Reply:
Q: What does generalized mean? 

If this means that it's equal in every other way than type, should we simplify equals to `return self.type == other.type and self.includes(other)` so the distiction becomes obvious?

Reply:
Judging from the code, I think it's supposed to mean: type is the same, and properties of one are subset of the other.